src/app/organizer/ItemTable.tsx

```diff
@@ -87,7 +87,7 @@ function mapStateToProps() {
   const itemsSelector = createSelector(
     allItemsSelector,
     searchFilterSelector,
-    (_, props: ProvidedProps) => props.categories,
+    (_: RootState, props: ProvidedProps) => props.categories,
```

Comment:
no implicit any should really disregard unused variables, that's annoying.

i actually really hate this being named _ when lodash's _ is used one line below it

Reply:
I personally prefer to use _ignored for this reason. It's a bit uglier but it wont clash with anything.

Reply:
I can rename it but yeah, it really cares that you add types to explicitly ignored stuff.

Reply:
Hmm, didn't know about this syntax: https://github.com/microsoft/TypeScript/issues/14154#issuecomment-280842441

I don't love the look of it but apparently `({}, props: ProvidedProps)` works because it's destructuring no variables... note that it only works if that parameter isn't undefined/null/not an object.

The other option is `(_state: unknown, props: ProvidedProps)` but at that point might as well just use the right type.